app/src/main/java/org/p2p/wallet/common/feature_toggles/toggles/remote/SettingsNetworkListFeatureToggle.kt

```diff
@@ -25,11 +27,17 @@ class SettingsNetworkListFeatureToggle(
         SettingsNetworkValue(url = "https://p2p.rpcpool.com", networkName = "mainnet-beta"),
         SettingsNetworkValue(url = "https://solana-api.projectserum.com", networkName = "mainnet-beta"),
         SettingsNetworkValue(url = "https://api.mainnet-beta.solana.com", networkName = "mainnet-beta")
-
     )
     override val value: List<SettingsNetworkValue>
         get() = valuesProvider.getString(featureKey)
             ?.let { gson.fromJsonReified<List<SettingsNetworkValue>>(it) }
             ?.takeIf(List<SettingsNetworkValue>::isNotEmpty)
             ?: defaultValue
+
+    fun getAvailableEnvironments(): List<NetworkEnvironment> {
```

Comment:
Move filtering available network to feature toggle, cause of some missed classes in core modul